### ISSUE TITLE
Clean up BASEA11yStrings

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#390]
There are some formatting issues, and many unused strings that should be removed.

**Comment by jessegreenberg** [comment]
Also, the 'String' suffix should be removed from all entries in the file,  that suffix isn't added entries in the json files.

**Comment by jessegreenberg** [comment]
Both of those points are done. Lets  reorganize the file now that we don't have  as much in there.

**Comment by jessegreenberg** [comment]
I added this TODO:
>TODO: 'near' and 'very close' landmarks strings  are now duplicated. Why?

**Comment by jessegreenberg** [comment]
This issue can be closed.

@terracoda @emily-phet, in case you are curious, about 255 strings were added for a11y.

**Comment by terracoda** [comment]
Cool! 255 is a nice number (https://en.wikipedia.org/wiki/255_(number)).
